packages/core/src/interpreter.ts

```diff
@@ -931,277 +879,36 @@ export class Interpreter<
       child.stop();
     }
   }
-  public spawn(entity: Spawnable, name: string, options?: SpawnOptions): Actor {
+  public spawn(
```

Comment:
This I'm not a particular fan of. I don't think we should encode support for different Spawnables in the Interpreter. It goes against composition principles and is not tree-shakeable.

Reply:
Good news: we can delete the entire thing!